internal/app/server/open_saves_test.go

```diff
@@ -148,6 +152,8 @@ func TestOpenSaves_RunServer(t *testing.T) {
 			t.Errorf("got err calling server.Run: %v", err)
 		}
 	})
+	// Sleep long enough to free up the connection
+	time.Sleep(serviceConfig.ShutdownGracePeriod)
```

Comment:
ditto

Reply:
same comment as above. If we do not sleep, we get an error when starting the next test since the port is still in use. I found that out from looking at the test logs in GCP.